requests-for-comments/2025-09-24-10x-engineering-onboarding.md

```diff
@@ -0,0 +1,88 @@
+# Request for comments: 10x-ing PostHog's engineering onboarding
+
+**Decision maker:** Engineering team leads + the blitzscale folks
+
+## Overview
+
+Our engineering onboarding is inconsistent and doesn't scale. Teams handle it in isolation, creating wildly different experiences that make it impossible to assess performance or teach our ways of working effectively.
+
+This RFC proposes cohort-based onboarding (HogBoarding) that standardizes the experience while building stronger cross-team connections.
+
+## Why now
+
+1. **We're hiring rapidly** and inconsistent onboarding amplifies all the problems
+2. **Recent onboarding failures** - several new hires haven't ramped effectively, leading to longer time-to-productivity and people leaving during probation
+3. **Internal poaching is breaking down** - we've moved people from overstaffed teams (Feature Flags) without communicating expectations upfront, creating confusion about hiring plans
+
+## Problem
+
+Current onboarding has too much variance in quality, operates in silos, doesn't teach the PostHog way effectively, creates an "onboarding tax" on high-performing teams, and makes inefficient use of our technical debt.
+
+**More details:**
+
+1. **Variance in quality.** Some teams have well-thought-out processes, others wing it. New hires get wildly different experiences depending on which team they join, making it impossible to assess performance consistently or ensure everyone learns our core ways of working.
+
+2. **Teams operate in silos.** New engineers only meet their immediate team members during their first weeks. This limits cross-team collaboration and means people miss out on learning from other parts of the product and codebase.
```

Comment:
+1 to meeting people

i go out of my way to arrange in-person onboarding in "hub" cities so that new hires meet other people

so we picked london or munich over my default of italy 😅 

i'd recommend the same in the US if we're not doing that